src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerCreateBundle.java

```diff
@@ -107,6 +107,7 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(
         caseData.setCaseBundles(bundlingService.buildBundleListValues(bundlingService.createBundle(bundleCallback)));
 
         caseData.setMultiBundleConfiguration(null);
+        caseData.setCaseDocuments(null);
```

Comment:
Any chance to cover this in unit test?

Reply:
Updated tests in `CaseworkerCreateBundleTest.shouldSuccessfullyCreateBundle()`